BuildResidentialHPXML/measure.rb

```diff
@@ -614,28 +614,28 @@ def arguments(model)
     arg = OpenStudio::Measure::OSArgument::makeDoubleArgument('neighbor_front_distance', true)
     arg.setDisplayName('Neighbor: Front Distance')
     arg.setUnits('ft')
-    arg.setDescription('The minimum distance between the simulated house and the neighboring house to the front (not including eaves). A value of zero indicates no neighbors.')
+    arg.setDescription('The minimum distance between the simulated unit and the neighboring unit to the front (not including eaves). A value of zero indicates no neighbors.')
```

Comment:
Seems like these should all say: "... between the simulated **unit** and the neighboring **building** ...". You are not describing the presence of an adjacent *unit* in a MF building (which obviously would have a distance of zero, but as the description states, zero indicates no neighbors). Those surfaces are adiabatic and have no windows and shading doesn't matter. Rather, you are specifying the presence of another *building* which might shade the unit's windows (and non-adiabatic walls).